org.pipoypipagames.cowsrevenge.json

```diff
@@ -0,0 +1,29 @@
+{
+  "app-id": "org.pipoypipagames.cowsrevenge",
```

Comment:
Do you own this domain name? It looks like it's available. You should control the path of your app-id. e.g. you could use com.github.dariasteam.cowsrevenge

Reply:
okay, it was certainly completely arbitrary